Tools/message_id.py

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
Do we need to maintain this numbering long term or can we generate it each build?